website/docs/r/vapp_network.html.markdown

```diff
@@ -58,6 +59,10 @@ The following arguments are supported:
 * `guest_vlan_allowed` (Optional) True if Network allows guest VLAN tagging. This value supported from vCD version 9.0
 * `static_ip_pool` - (Optional) A range of IPs permitted to be used as static IPs for virtual machines; see [IP Pools](#ip-pools) below for details.
 * `dhcp_pool` - (Optional) A range of IPs to issue to virtual machines that don't have a static IP; see [IP Pools](#ip-pools) below for details.
+* `org_network` - (Optional; *v2.7+*) An Org network name to which vApp network is connected to. If not configured, then isolated network created.
+* `firewall_enabled` - (Optional; *v2.7+*) Firewall service enabled or disabled. Default - true.
```

Comment:
Polishing. Please adjust the format of the "defaults" sentences in this PR for consistency with other docs. We can do like this:
```suggestion
* `firewall_enabled` - (Optional; *v2.7+*) Firewall service enabled or disabled. Default is `true`.
```

Reply:
done